prime

```diff
@@ -0,0 +1,22 @@
+#include<iosteam>
```

Comment:
@JHAGOVIND Have you tried running the code. It is missing .cpp extension. Before creating a PR please ensure that coding is running fine